src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/TemplateMetadata.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\FormMetadata;
+
+class TemplateMetadata
+{
+    protected ?string $controller = null;
+
+    protected ?string $view = null;
+
+    protected ?CacheLifetimeMetadata $cacheLifetime = null;
+
+    public function __construct()
+    {
+    }
+
+    public function getController(): ?string
+    {
+        return $this->controller;
+    }
+
+    public function setController(?string $controller): void
+    {
+        $this->controller = $controller;
+    }
+
+    public function getView(): ?string
+    {
+        return $this->view;
+    }
+
+    public function setView(?string $view): void
+    {
+        $this->view = $view;
+    }
+
+    public function getCacheLifetime(): ?CacheLifetimeMetadata
```

Comment:
🤔 I’m not sure how I feel about this, on one hand, I would expect `getCacheLifetime` to return a scalar value directly, not an object. On the other hand, `getCacheLifetimeMetadata` is quite a long method name.

Reply:
cachelifeime does have 2 types:

```xml
    <cacheLifetime type="expression">0 8 * * *</cacheLifetime>
```


```xml
    <cacheLifetime type="seconds">2400</cacheLifetime>
```

The [cachelifetimeresolver](https://github.com/sulu/sulu/blob/33e8b72844761ffc7483a3a95649445c0f48c424/src/Sulu/Bundle/HttpCacheBundle/CacheLifetime/CacheLifetimeResolver.php#L20) is responsible to convert the string value into a integer

Reply:
did choice `getCacheLifetimeMetadata` as it is also `getSchema` not `getSchemaMetadata` method, think the idea behind is that all inside that namespace is automatically Metadata.

Reply:
Yes that sounds reasonable 👍  thanks for the explanation